source/github-helpers/get-default-branch.ts

```diff
@@ -14,7 +14,11 @@ const branchInfoRegex = /([^ ]+)\.$/;
 
 export default cache.function(async (repository: Partial<RepositoryInfo> = getRepositoryInfo()): Promise<string> => {
 	if (JSON.stringify(repository) === JSON.stringify(getRepositoryInfo())) {
-		if (!isForkedRepo()) {
+		if (pageDetect.isRepoRoot() && !String(getRepoPath()).startsWith('tree/')) {
```

Comment:
Isn't this equivalent?
```suggestion
		if (pageDetect.isRepoRoot() && !pageDetect.isRepoTree()) {
```

Reply:
>Note: isRepoTreecannot be used instead of !String(getRepoPath()).startsWith('tree/') since isRepoTree includes isRepoRoot

Reply:
> No its not. I left a note in the first post

I will have to read your first post more carefully.

Reply:
Can you leave those comments inline instead of in the first post? So they're linked exactly to the comment

Reply:
Example: https://github.com/sindresorhus/refined-github/pull/3206#pullrequestreview-427166121

Reply:
Sure